### PR TITLE
Replace owl:Thing with rdfs:Class

### PR DESCRIPTION
**Pull request by OR13**
Remove OWL from security vocab.

### CODE REVIEW

**Comment by iherman** [comment]
What does "parent Class" mean, and why do we need that? Isn't it clearer if that is replaced by saying 'of type rdfs:Class'?

**Comment by OR13** [comment]
happy to make additional changes here.. @dlongley @msporny thoughts?

**Comment by msporny** [comment]
> What does "parent Class" mean, and why do we need that? Isn't it clearer if that is replaced by saying 'of type rdfs:Class'?

This is another one of those things that's over a decade old. I don't have a strong opinion on keeping "Parent Class"... honestly, I'm not sure it even matters to list it. "Of Type" would be a strange header when rendered. We could do "Subclass of" or "Type of"... again, no strong preference.

**Comment by iherman** [comment]
> > What does "parent Class" mean, and why do we need that? Isn't it clearer if that is replaced by saying 'of type rdfs:Class'?
> 
> This is another one of those things that's over a decade old. I don't have a strong opinion on keeping "Parent Class"... honestly, I'm not sure it even matters to list it. "Of Type" would be a strange header when rendered. We could do "Subclass of" or "Type of"... again, no strong preference.

I do not have an opinion on how it should appear editorially. From an RDF point of view, if we define a vocabulary that includes a class, what I am looking for is a triple of the form 

```
EcdsaSecp256k1Signature2019 rdf:type rdfs:Class
```

See, among the many examples, the [web annotation vocabulary](https://www.w3.org/ns/oa.jsonld). The term "Parent Class" is unclear and not a usual terminology in RDF vocabulary definitions and I would prefer to drop this terminology altogether. Stating somewhere clearly, in the English text, that this is a class seems to be perfectly fine to me.

**Comment by OR13** [comment]
Given that the vocabulary is already seperated into `Classes` and properties, I would also be in favor of removing "Parent Class" entirely.

File: index.html [hunk 1]
```diff
@@ -343,7 +330,7 @@ <h3>EncryptedMessage</h3>
       </section>
 
       <section id="GraphSignature2012" about="https://w3id.org/security#GraphSignature2012"
-        typeof="rdfs:Class">
+        typeof="sec:Signature">
```
Comment (OR13) [inline index.html:333]: @msporny @dlongley @davidlehn @iherman is this kind of annotation actually useful?

**Comment by OR13** [comment]
@msporny I made some changes since you reviewed.

**Comment by OR13** [comment]
I will merge this in a few days without objection.

**Pull request merged**